Tests/ManualTests/ManualTestingApp/PointerAndMouse/PointerAndMouse.ux.uno

```diff
@@ -53,4 +53,3 @@ public partial class PointerAndMouse
         return result;
     }
 }
-
```

Comment:
Yeah. It's something my editor did by default, and I considered dropping it. I decided to leave it in, because this is on master where whitespace-cleanups aren't generally a problem. If it was a release-branch, I'd be more skeptical. Having two newlines at the end of a file seems just silly.

That being said, I'm fine with dropping it if people care.